### PR TITLE
I don't know what I am doing...

### PR DESCRIPTION
**Pull request by joel-labbe**
Committing some test code I copied into PyCharm. I am also testing how to branch and merge from master so bare with me.

### CODE REVIEW

File: camera_interface.py [hunk 1]
```diff
@@ -0,0 +1,54 @@
+# from gpiozero import MotionSensor
+from time import sleep
+
+from picamera import PiCamera
```
Comment (FanchenBao) [inline camera_interface.py]: Is `picamera` a dependency module?

**Comment by joel-labbe** [comment]
I s the formatting good? I let pycharm do its thing, but I am not sure it is PEP8.

**Comment by FanchenBao** [comment]
Looks good. As long as PyCharm doesn't complain, it's generally fine. I left a comment on your code.

File: camera_interface.py (hunk 1 above)
Comment (FanchenBao) [reply camera_interface.py]: @joel-labbe This is the comment. I forgot to publish it earlier.

Comment (FanchenBao) [reply camera_interface.py]: Okay. Just to remind you that all dependencies must be mentioned in the readme, along with how they shall be installed.

**Pull request merged**